core/src/main/java/io/parsingdata/metal/Shorthand.java

```diff
@@ -16,58 +16,18 @@
 
 package io.parsingdata.metal;
 
-import io.parsingdata.metal.expression.value.reference.Len;
-
 import io.parsingdata.metal.encoding.Encoding;
 import io.parsingdata.metal.expression.Expression;
 import io.parsingdata.metal.expression.True;
-import io.parsingdata.metal.expression.comparison.ComparisonExpression;
-import io.parsingdata.metal.expression.comparison.Eq;
-import io.parsingdata.metal.expression.comparison.EqNum;
-import io.parsingdata.metal.expression.comparison.EqStr;
-import io.parsingdata.metal.expression.comparison.GtNum;
-import io.parsingdata.metal.expression.comparison.LtNum;
-import io.parsingdata.metal.expression.logical.And;
-import io.parsingdata.metal.expression.logical.BinaryLogicalExpression;
-import io.parsingdata.metal.expression.logical.Not;
-import io.parsingdata.metal.expression.logical.Or;
-import io.parsingdata.metal.expression.logical.UnaryLogicalExpression;
-import io.parsingdata.metal.expression.value.BinaryValueExpression;
-import io.parsingdata.metal.expression.value.Cat;
-import io.parsingdata.metal.expression.value.Const;
-import io.parsingdata.metal.expression.value.ConstantFactory;
-import io.parsingdata.metal.expression.value.Elvis;
-import io.parsingdata.metal.expression.value.FoldLeft;
-import io.parsingdata.metal.expression.value.FoldRight;
-import io.parsingdata.metal.expression.value.Reducer;
-import io.parsingdata.metal.expression.value.UnaryValueExpression;
-import io.parsingdata.metal.expression.value.Value;
-import io.parsingdata.metal.expression.value.ValueExpression;
-import io.parsingdata.metal.expression.value.arithmetic.Add;
-import io.parsingdata.metal.expression.value.arithmetic.Div;
-import io.parsingdata.metal.expression.value.arithmetic.Mod;
-import io.parsingdata.metal.expression.value.arithmetic.Mul;
-import io.parsingdata.metal.expression.value.arithmetic.Neg;
+import io.parsingdata.metal.expression.comparison.*;
```

Comment:
Yes. It also has a different import ordering. I suppose we can fix this but I'm fine with slowly migrating to IntelliJ default.